src/main/java/com/getify/minify/Minify.java

```diff
@@ -0,0 +1,103 @@
+//
+//  Minify.java
+//  jsonminifier - Ported from https://github.com/getify/JSON.minify
+//
+//  Created by Bernhard Gass on 8/01/13.
+//  Copyright © 2013 Bernhard Gass. All rights reserved.
+
+package com.getify.minify;
```

Comment:
`io.github.getify` should work fine then. I'll get that changed and we can get this PR merged.